test/unit_tests/test_interrogate_setup.py

```diff
@@ -153,4 +169,14 @@ def test_interrogate_setup_py(self):
 set(foo_SETUP_PY_MODULE_DIRS "")""", contents)
             os.remove(outfile)
         finally:
-            shutil.rmtree(rootdir)
+            shutil.rmtree(develdir)
+            shutil.rmtree(srcdir)
+            os.chdir(curdir)
```

Comment:
The finally block should first change the cwd and then try to remove the other folders (otherwise on e.g. Windows the cwd will prevent deleting the cwd `srcdir`).